src/components/interface/Header/ToolTranslate.js

```diff
@@ -30,7 +33,8 @@ const ToolTranslate = ({
         >
           {currentLang}
         </button>
-        <div className={`fr-translate__menu fr-menu ${isOpen ? 'fr-collapse--expanded' : 'fr-collapse'}`} id="translate-516">
+        {/* eslint-disable-next-line jsx-a11y/no-static-element-interactions */}
```

Comment:
Can you resolve the eslint issue instead ? 
I think putting the div as a role menuitem should do the trick ?